src/executor/admin/SpaceExecutor.cpp

```diff
@@ -64,6 +65,11 @@ folly::Future<Status> DescSpaceExecutor::execute() {
                 row.values.emplace_back(properties.get_charset_name());
                 row.values.emplace_back(properties.get_collate_name());
                 row.values.emplace_back(SchemaUtil::typeToString(properties.get_vid_type()));
+                if (properties.__isset.group_name) {
+                    row.values.emplace_back(properties.get_collate_name());
```

Comment:
Why to check `group_name` while accessing `collate_name`?

Reply:
Good catch